onnxruntime/core/external_ops/pyop.h

```diff
@@ -0,0 +1,209 @@
+#pragma once
```

Comment:
please put the code in a cc file. header should only contain interfaces.

Reply:
Done.